scoreboard/serializers.py

```diff
@@ -108,7 +110,7 @@ class GameSerializer(serializers.ModelSerializer):
     wallclock = serializers.SerializerMethodField()
 
     class Meta:
-        model = GameRecord
+        model = Game
```

Comment:
by changing the model here from `GameRecord` to `Game` we break the serializer, because the fields no longer match the model fields.
i think the right way to go might actually be having a separate `GameRecordSerlializer` and `GameSerializer`, and additionally some read-only Serializers for views that expand the GameRecord field of Game into the displayed data,
i.e. showing not: `{ "player": "aoei", "source": "hdf-eu", "record": "{some-uuid}" }`, but rather
`{ "player": "aoei", "source": "hdf-eu", "race": "dwa", "role": "val", "death": "killed by a missing unit test" }`